weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CompareEvaluationsPage/sections/ExampleCompareSection/exampleCompareSectionUtil.ts

```diff
@@ -130,50 +141,40 @@ export const useFilteredAggregateRows = (state: EvaluationComparisonState) => {
       Object.values(rowCollection.evaluations).forEach(modelCollection => {
         Object.values(modelCollection.predictAndScores).forEach(
           predictAndScoreRes => {
-            const datasetRow =
-              state.loadableComparisonResults.result?.inputs[
-                predictAndScoreRes.rowDigest
-              ];
-            if (datasetRow != null) {
-              const output = predictAndScoreRes._rawPredictTraceData?.output;
-              rows.push({
-                id: predictAndScoreRes.callId,
-                evaluationCallId: predictAndScoreRes.evaluationCallId,
-                inputDigest: datasetRow.digest,
-                inputRef: predictAndScoreRes.exampleRef,
-                input: flattenObjectPreservingWeaveTypes({
-                  input: datasetRow.val,
-                }),
-                output: flattenObjectPreservingWeaveTypes({output}),
-                scores: Object.fromEntries(
-                  [...Object.entries(state.summary.scoreMetrics)].map(
-                    ([scoreKey, scoreVal]) => {
-                      return [
-                        scoreKey,
-                        resolveScoreMetricValueForPASCall(
-                          scoreVal,
-                          predictAndScoreRes
-                        ),
-                      ];
-                    }
-                  )
-                ),
-                path: [
-                  rowDigest,
-                  predictAndScoreRes.evaluationCallId,
-                  predictAndScoreRes.callId,
-                ],
-                predictAndScore: predictAndScoreRes,
-              });
-            }
+            const output = predictAndScoreRes._rawPredictTraceData?.output;
+            rows.push({
+              id: predictAndScoreRes.callId,
+              evaluationCallId: predictAndScoreRes.evaluationCallId,
+              inputDigest: predictAndScoreRes.rowDigest,
+              inputRef: predictAndScoreRes.exampleRef,
+              output: flattenObjectPreservingWeaveTypes({output}),
+              scores: Object.fromEntries(
```

Comment:
in @andrewtruong's upcoming PR, we can't count that the data comes from a dataset (gasp!) it might be good to at least leave a comment here as this would be a possible location to record the raw predict_and_score inputs as the presumed data row.